telehealth-example/src/main/java/com/pubnub/components/example/telehealth/ui/view/Login.kt

```diff
@@ -0,0 +1,232 @@
+                        .size(width = 420.dp, height = 48.dp)
+                        .padding(start = 24.dp, end = 24.dp),
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    Button(
+                        modifier = Modifier.fillMaxSize(),
+                        onClick = {
+                            visible = tryToLogin(loginViewModel, login, context)
+                        },
+                        shape = RoundedCornerShape(20),
+                        colors = ButtonDefaults.buttonColors(
+                            backgroundColor = HyperLinkColor
+                        )
+                    ) {
+                        Text(
+                            text = stringResource(R.string.login),
+                            style = Typography.body2,
+                        )
+                    }
+                }
+                AnimatedVisibility(
+                    visible = visible,
+                    enter = fadeIn(
+                        initialAlpha = 0.4f
+                    ),
+                    exit = fadeOut(
+                        animationSpec = tween(durationMillis = 250)
+                    )
+                ) {
+                    Row(modifier = Modifier.padding(top = 12.dp, start = 24.dp)) {
+                        Image(
+                            painter = painterResource(id = R.drawable.ic_icon),
+                            contentDescription = stringResource(id = R.string.error_icon),
+                            modifier = Modifier.padding(top = 4.dp)
+                        )
+                        Text(
+                            text = stringResource(R.string.password_error),
+                            style = TextStyle(
+                                fontSize = 16.sp,
+                                color = Color.hsl(2F, 0.72F, 0.53F, 1F)
+                            ),
+                            modifier = Modifier.padding(start = 8.dp)
+                        )
+                    }
+                }
+
+            }
+            Box(
+                modifier = Modifier
+                    .background(LoginInfoBoxBackgroundColor)
+                    .align(
+                        Alignment.BottomCenter
+                    )
+            ) {
+                HyperlinkText(
+                    modifier = Modifier
+                        .padding(top = 40.dp, start = 72.dp, end = 72.dp, bottom = 30.dp),
+                    fullText = stringResource(R.string.password_info),
+                    hyperlink = Hyperlink("Demo page", "https://github.com/pubnub/chat-components-android-examples/blob/telehealth-example/telehealth-example/README.md")
+                )
+            }
+        }
+    }
+
+    private fun tryToLogin(loginViewModel: LoginViewModel, login: String, context: Context): Boolean{
```

Comment:
The returned value suggests whether the method (login) was successful. In fact, it is a negation of this value.

Reply:
Btw, it should be a method inside loginViewModel